### PR TITLE
convert ShareContactsViewController & PermissionDeniedViewController to Swift

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

convert `ShareContactsViewController` & `PermissionDeniedViewController` to Swift

### CODE REVIEW

File: Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    
+    private var initialConstraintsCreated = false
+    private let heroLabel: UILabel = UILabel.createHeroLabel()
+    private var settingsButton: Button!
+    private var laterButton: UIButton!
+    private let backgroundBlurView: UIVisualEffectView = UIVisualEffectView.createBackgroundBlurView()
+
+    private var monochromeStyle = false
+
+    class func addressBookAccessDeniedViewController(withMonochromeStyle monochromeStyle: Bool) -> PermissionDeniedViewController {
```
Comment (typfel) [inline Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift]: Pretty sure this monochrome style is not used anymore. Did you check if this ever called with `monochromeStyle == true`?

File: Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        
+        updateViewConstraints()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
```
Comment (typfel) [inline Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift]: Can be deleted

File: Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift (hunk 1 above)
Comment (billypchan) [reply Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift]: Yes, it is always false now.

File: Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        vc.heroLabel.attributedText = attributedText
+        
+        vc.settingsButton.setTitle("registration.address_book_access_denied.settings_button.title".localized.uppercased(), for: .normal)
+        
+        vc.laterButton.setTitle("registration.address_book_access_denied.maybe_later_button.title".localized.uppercased(), for: .normal)
+
+        return vc
+    }
+    
+    class func push() -> PermissionDeniedViewController {
```
Comment (typfel) [inline Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift]: This name is a bit out of context now, I see where it comes from but it doesn't make much sense anymore. It looks like this class method is creating a default instance of the `PermissionDeniedViewController`. Maybe we can call it something like `default`

Comment (billypchan) [reply Wire-iOS/Sources/Permissions/PermissionDeniedViewController.swift]: restored to original objc name `pushDeniedViewController`

File: Wire-iOS/Sources/UserInterface/ContactsUI/ContactsViewController+ShareContactsViewControllerDelegate.swift [hunk 4]
```diff
@@ -1,4 +1,4 @@
-//
+
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/ContactsUI/ContactsViewController+ShareContactsViewControllerDelegate.swift]: Extra space added

**Pull request merged**